package.json

```diff
@@ -1,18 +1,24 @@
 {
   "scripts": {
     "test": "sass-lint static/**/*.scss --verbose --no-exit",
-    "build": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace static/css/styles.css && postcss static/css/styles.css --use cssnano > static/css/styles.min.css",
+    "build": "yarn run build-js && yarn run build-css",
+    "build-css": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace static/css/styles.css && postcss static/css/styles.css --use cssnano > static/css/styles.min.css",
+    "build-js": "cp node_modules/d3/build/d3.min.js static/js/modules && cp node_modules/topojson-client/dist/topojson-client.min.js static/js/modules",
     "watch": "watch -p 'static/sass/*.scss' -c 'yarn run build'",
     "clean": "rm -rf node_modules yarn-error.log static/css *.log *.sqlite _site/ build/ .jekyll-metadata"
   },
   "author": "Canonical webteam",
   "devDependencies": {
     "autoprefixer": "^7.1.3",
+    "cssnano": "^3.10.0",
     "node-sass": "^4.5.3",
     "postcss-cli": "^4.1.0",
     "sass-lint": "^1.10.2",
     "vanilla-framework": "1.5.2",
-    "watch-cli": "^0.2.2",
-    "cssnano": "^3.10.0"
+    "watch-cli": "^0.2.2"
+  },
+  "dependencies": {
+    "d3": "^4.10.2",
```

Comment:
As these still won't get packaged up with the site directly (they get copied to the static folder and packaged from there) they're really just as much devDependencies as the rest of the stuff. But in other projects we simply use `dependencies`, as there's not really a material difference for our sites (they are never included as modules in other projects, which is when the distinction matters). Could you just put everything under `dependencies`?